theme/less/_content.less

```diff
@@ -2,3 +2,31 @@ main article img
 {
   max-width: 100%;
 }
+
+article
+{
+  color: #444444;
+}
+
+article p, article ul, article ol
+{
+  font-size: 14px;
+  margin: 0 0 20px 0;
```

Comment:
Detail: Wonder if we want 20px bottom margin on the last element? that might provide unexpected results in general, I prefer using top margin if top most item in content (i.e. h1) has big margin bottom, in that case the margins might get merged, but not sure if it's same in our case.

This might not be a big deal though.. Leaving as "detail".

Reply:
You're right the top margins of  `p`  are collapsing  with bigger margin above. Also default margin or `p` is 1rem which is in most cases equal to 16px by (default browser settings). So this if this property will be removed nothing valuable will be changed.

Reply:
> Also default margin of p is 1rem

Is it? I wouldn't rely on it, usually it's up for the browser vendors to specify those. 

I found so far that users are sometimes refering to [this resource](https://www.w3.org/TR/CSS21/sample.html), but as the page says is "informative, not normative".